web/src/copy_messages.ts

```diff
@@ -264,8 +279,11 @@ export function copy_handler(): boolean {
     // Select div so that the browser will copy it
     // instead of copying the original selection
     insert_and_select_div($div, selection);
```

Comment:
We don’t need to insert the div into the body, nor alter the user’s selection, just to get the HTML and the text.